lib/route/src/Obelisk/Route.hs

```diff
@@ -200,6 +205,48 @@ mapSome f (Some a) = Some $ f a
 hoistR :: (forall x. f x -> g x) -> R f -> R g
 hoistR f (x :=> Identity y) = f x :/ y
 
+--------------------------------------------------------------------------------
+-- Dealing with pairs (i.e. non-dependently-typed subroutes/paths)
+--------------------------------------------------------------------------------
+
+infixr 5 :.
+type (:.) = (,)
```

Comment:
What value does this bring over just using tuples?

Reply:
It looks like it's only used in one place?

Reply:
When people use it right-associatively, it looks nicer than nested tuples.  Without something like this, people will use tuples with more entries than pairs, which can't be broken down very easily by other primitives.

Reply:
Ah makes sense thanks.